diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/chapter3/files/NewlinesRule.kt

```diff
@@ -342,6 +384,29 @@ class NewlinesRule(configRules: List<RulesConfig>) : DiktatRule("newlines", conf
             }
         }
 
+    private fun handleValueParameterList(node: ASTNode) {
```

Comment:
How is this method different from `handleValueParameterList` declared just below it?

Reply:
This method is not used. I will remove it